cves/CVE-2014-8109.yml

```diff
@@ -4,7 +4,7 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-305 Authentication Bypass by Primary Weakness
```

Comment:
Please keep it to just the name cwe-305 or 305

Reply:
I've updated this and the other CVE I worked on (CVE-2014-0094 Apache Struts) to remove the unnecessary description of the CWE.